spring-data-microservice-transaction-sample/README.md

```diff
@@ -465,84 +457,76 @@ if (updatedCreditTotal > customer.creditLimit) {
 
 ```java
-Collections.singletonList(
-  RemotePrepareCommitPhaseOperations.createSerializable(
-    this::callPrepareEndpoint,
-    this::callValidateEndpoint,
-    this::callCommitEndpoint,
-    this::callRollbackEndpoint
+TwoPcResult<PlaceOrderResponse> result = orderRepository.executeTwoPcTransaction(txId ->{
+    ...
+  },
+
+  Collections.singletonList(
+    RemotePrepareCommitPhaseOperations.createSerializable(
+      this::callPrepareEndpoint,
+      this::callValidateEndpoint,
+      this::callCommitEndpoint,
+      this::callRollbackEndpoint
+    )
   )
-)
+);
 ```
 
 ![Sequence Diagram of High Level 2PC API](images/seq-diagram-high-level-2pc-api.png)
 
-In the `prepare` endpoint of the Customer Service, it resumes and prepares the transaction (the code is [here](customer-service/src/main/java/sample/customer/CustomerService.java#L122-L126)):
+In the `prepare` endpoint of the Customer Service, it resumes and prepares the transaction with `ScalarDbTwoPcRepository.prepareTransactionOnParticipant()`. For reference, see [`CustomerService.java`](customer-service/src/main/java/sample/customer/CustomerService.java).
 
 ```java
-execTwoPcOperation(request.getTransactionId(), false, responseObserver, "Payment", () -> {
-  // Prepare the transaction
-  customerRepository.prepare();
-});
+customerRepository.prepareTransactionOnParticipant(request.getTransactionId());
 ```
 
-The transaction is resumed in `execTwoPcOperation()` as shown above.
-
-
-In the `validate` endpoint of the Customer Service, the microservice resumes and validates the transaction (the code is [here](customer-service/src/main/java/sample/customer/CustomerService.java#L131-L135)):
+In the `validate` endpoint of the Customer Service, it resumes and validates the transaction with `ScalarDbTwoPcRepository.validateTransactionOnParticipant()`. For reference, see [`CustomerService.java`](customer-service/src/main/java/sample/customer/CustomerService.java).
 
 ```java
-execTwoPcOperation(request.getTransactionId(), false, responseObserver, "Validate", () -> {
-  // Validate the transaction
-  customerRepository.validate();
-  return Empty.getDefaultInstance();
-});
+customerRepository.validateTransactionOnParticipant(request.getTransactionId());
 ```
 
-In the `commit` endpoint of the Customer Service, the microservice resumes and commits the transaction (the code is [here](customer-service/src/main/java/sample/customer/CustomerService.java#L140-L144)):
+In the `commit` endpoint of the Customer Service, it resumes and commits the transaction with `ScalarDbTwoPcRepository.commitTransactionOnParticipant()`. For reference, see [`CustomerService.java`](customer-service/src/main/java/sample/customer/CustomerService.java).
```

Comment:
Does `it` mean ``the `commit` endpoint``? If so, how about we change this sentence as follows?:

```suggestion
In the `commit` endpoint of the Customer Service, the endpoint resumes and commits the transaction by using `ScalarDbTwoPcRepository.commitTransactionOnParticipant()`. For reference, see [`CustomerService.java`](customer-service/src/main/java/sample/customer/CustomerService.java).
```